test/integration/javaclasses/PrefixFilter.js

```diff
@@ -15,18 +15,20 @@
  */
 'use strict';
 
+const TestUtil = require('../../TestUtil');
+
 function PrefixFilter(prefix) {
     this.prefix = prefix;
     this.factoryId = 66;
     this.classId = 4;
 }
 
 PrefixFilter.prototype.readData = function (inp) {
-    this.prefix = inp.readString();
+    this.prefix = TestUtil.readStringFromInput(inp);
```

Comment:
Why change these other than the ones in backward_compatible folder

Reply:
Well, these files are used in the backward compatible test suite.

Reply:
Thanks for explaining